cmd/psi.go

```diff
@@ -358,15 +369,23 @@ func checkPsiCPUPressure(config *psiConfig) result.PartialResult {
 	return cpuCheck
 }
 
-//nolint:funlen,gocognit
+//nolint:funlen,gocognit,gocyclo
```

Comment:
I'd prefer this in the `.golangci.yml` since we got so many in this file.

Reply:
done